readme.md

```diff
@@ -5,7 +5,7 @@
 
 <br>
 
-visit [jotai.org](https://jotai.org) or `npm i jotai`
```

Comment:
At least, this was very intentional.

Reply:
Yes, let's keep the shorthand here at least. 🙂